README.md

```diff
@@ -155,13 +155,14 @@ ESLint follows [semantic versioning](http://semver.org). However, due to the nat
 * Minor release (might break your lint build)
     * A bug fix in a rule that results in ESLint reporting more errors.
     * A new rule is created.
-    * A new option to an existing rule is created.
+    * A new option to an existing rule is created, not reporting more errors.
     * An existing rule is deprecated.
     * A new CLI capability is created.
     * New capabilities to the public API are added (new classes, new methods, new arguments to existing methods, etc.).
     * A new formatter is created.
 * Major release (likely to break your lint build)
     * `eslint:recommended` is updated.
+    * A new option to an existing rule is created, reporting more errors.
```

Comment:
A new option to an existing rule that results in ESLint reporting more errors by default